src/main/java/com/snowflake/kafka/connector/SnowflakeSinkTask.java

```diff
@@ -421,21 +463,44 @@ private KafkaRecordErrorReporter createKafkaRecordErrorReporter() {
                   errantRecordReporter.report(record, error).get();
                 } catch (InterruptedException | ExecutionException e) {
                   final String errMsg = "ERROR reporting records to ErrantRecordReporter";
-                  LOGGER.error(errMsg, e);
+                  this.DYNAMIC_LOGGER.error(errMsg, e);
                   throw new ConnectException(errMsg, e);
                 }
               };
         } else {
-          LOGGER.info("Errant record reporter is not configured.");
+          this.DYNAMIC_LOGGER.info("Errant record reporter is not configured.");
         }
       } catch (NoClassDefFoundError | NoSuchMethodError e) {
         // Will occur in Connect runtimes earlier than 2.6
-        LOGGER.info("Kafka versions prior to 2.6 do not support the errant record reporter.");
+        this.DYNAMIC_LOGGER.info(
+            "Kafka versions prior to 2.6 do not support the errant record reporter.");
       }
     }
     return result;
   }
 
+  private String getTaskLoggingTag() {
```

Comment:
move this to logger handler?

Reply:
answered in earlier comment, resolving this one to keep one thread